recipe_scrapers/joshuaweissman.py

```diff
@@ -0,0 +1,101 @@
+# mypy: allow-untyped-defs
+
+# from pprint import pprint
+from ._abstract import AbstractScraper
+from ._grouping_utils import group_ingredients
+from ._utils import get_minutes, get_yields, normalize_string
+
+
+class JoshuaWeissman(AbstractScraper):
+    @classmethod
+    def host(cls):
+        return "joshuaweissman.com"
+
+    def author(self):
+        return self.schema.author()
+
+    def title(self):
+        return self.schema.title()
+
+    def category(self):
+        return self.schema.category()
+
+    def total_time(self):
+        # Get all spans and get their text
+        spans = self.soup.findAll(name="span")
+        spans_texts = [tag.get_text().lower() for tag in spans]
+        # Filter out all spans that don't contain time
+        time_texts = [tag for tag in spans_texts if "time" in tag]
+
+        # Find first string that contains the text
+        def check_text(check_string):
+            return next(
+                (string for string in time_texts if check_string in string), None
+            )
+
+        total = check_text("total time")
+        if total:
+            return get_minutes(total)
+        prep = check_text("prep time")
+        if prep:
+            prep = get_minutes(prep)
+        cook = check_text("cook time")
+        if cook:
+            cook = get_minutes(cook)
+        return prep + cook
```

Comment:
This might not be necessary, but as a safety guard against zero-valued times here:

```suggestion
        if prep + cook:
            return prep + cook
```

(similarly it could make sense to add a similar condition for the `total` return case)